daemon/src/position_metrics.rs

```diff
@@ -0,0 +1,301 @@
+use crate::db;
+use async_trait::async_trait;
+use futures::StreamExt;
+use model::CfdEvent;
+use model::EventKind;
+use model::Position;
+use model::Usd;
```

Comment:
Note on modularity: If the `db` ends up being extracted as a separate crate, this actor could also be a separate one.

The compile-times of the workspace will continue growing if we continue to just add stuff to `daemon` and not try and modularise it into individual crates.